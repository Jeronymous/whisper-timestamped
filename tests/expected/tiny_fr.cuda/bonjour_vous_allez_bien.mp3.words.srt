1
00:00:00,460 --> 00:00:01,300
Bonjour!

2
00:00:01,860 --> 00:00:02,140
Est-ce

3
00:00:02,140 --> 00:00:02,240
que

4
00:00:02,240 --> 00:00:02,340
vous

5
00:00:02,340 --> 00:00:02,480
allez

6
00:00:02,480 --> 00:00:03,060
bien?

7
00:00:33,000 --> 00:00:33,640
Bonjour!

8
00:00:34,920 --> 00:00:35,020
Bonjour!

9
00:00:35,020 --> 00:00:35,120
Est-ce

10
00:00:35,120 --> 00:00:35,220
que

11
00:00:35,220 --> 00:00:35,320
vous

12
00:00:35,320 --> 00:00:35,420
allez

13
00:00:35,420 --> 00:00:35,580
bien?

