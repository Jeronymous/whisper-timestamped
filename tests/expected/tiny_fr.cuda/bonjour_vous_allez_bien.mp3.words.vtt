WEBVTT

00:00.460 --> 00:01.300
Bonjour!

00:01.860 --> 00:02.140
Est-ce

00:02.140 --> 00:02.240
que

00:02.240 --> 00:02.340
vous

00:02.340 --> 00:02.480
allez

00:02.480 --> 00:03.060
bien?

00:33.000 --> 00:33.640
Bonjour!

00:34.920 --> 00:35.020
Bonjour!

00:35.020 --> 00:35.120
Est-ce

00:35.120 --> 00:35.220
que

00:35.220 --> 00:35.320
vous

00:35.320 --> 00:35.420
allez

00:35.420 --> 00:35.580
bien?

