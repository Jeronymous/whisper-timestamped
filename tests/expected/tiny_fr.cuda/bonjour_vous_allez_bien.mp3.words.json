{
  "text": " Bonjour! Est-ce que vous allez bien? Bonjour! Bonjour! Est-ce que vous allez bien?",
  "segments": [
    {
      "id": 0,
      "seek": 0,
      "start": 0.46,
      "end": 1.3,
      "text": " Bonjour!",
      "tokens": [
        25431,
        2298
      ],
      "temperature": 0.0,
      "avg_logprob": -0.774240239461263,
      "compression_ratio": 0.8181818181818182,
      "no_speech_prob": 0.04296562075614929,
      "words": [
        {
          "text": "Bonjour!",
          "start": 0.46,
          "end": 1.3
        }
      ]
    },
    {
      "id": 1,
      "seek": 148,
      "start": 1.86,
      "end": 3.06,
      "text": " Est-ce que vous allez bien?",
      "tokens": [
        50364,
        4410,
        12,
        384,
        631,
        2630,
        18146,
        3610,
        2506,
        50464
      ],
      "temperature": 0.0,
      "avg_logprob": -0.48053342645818536,
      "compression_ratio": 0.7714285714285715,
      "no_speech_prob": 0.04136796295642853,
      "words": [
        {
          "text": "Est-ce",
          "start": 1.86,
          "end": 2.14
        },
        {
          "text": "que",
          "start": 2.14,
          "end": 2.24
        },
        {
          "text": "vous",
          "start": 2.24,
          "end": 2.34
        },
        {
          "text": "allez",
          "start": 2.34,
          "end": 2.48
        },
        {
          "text": "bien?",
          "start": 2.48,
          "end": 3.06
        }
      ]
    },
    {
      "id": 2,
      "seek": 3148,
      "start": 33.0,
      "end": 33.64,
      "text": " Bonjour!",
      "tokens": [
        25431,
        2298
      ],
      "temperature": 0.0,
      "avg_logprob": -0.3400673548380534,
      "compression_ratio": 0.8181818181818182,
      "no_speech_prob": 0.5503692030906677,
      "words": [
        {
          "text": "Bonjour!",
          "start": 33.0,
          "end": 33.64
        }
      ]
    },
    {
      "id": 3,
      "seek": 3348,
      "start": 34.92,
      "end": 35.02,
      "text": " Bonjour!",
      "tokens": [
        25431,
        2298
      ],
      "temperature": 0.0,
      "avg_logprob": -0.2368337631225586,
      "compression_ratio": 0.8181818181818182,
      "no_speech_prob": 0.40514007210731506,
      "words": [
        {
          "text": "Bonjour!",
          "start": 34.92,
          "end": 35.02
        }
      ]
    },
    {
      "id": 4,
      "seek": 3448,
      "start": 35.02,
      "end": 35.58,
      "text": " Est-ce que vous allez bien?",
      "tokens": [
        50364,
        4410,
        12,
        384,
        631,
        2630,
        18146,
        3610,
        2506,
        50464
      ],
      "temperature": 0.0,
      "avg_logprob": -0.1729703816500577,
      "compression_ratio": 0.7714285714285715,
      "no_speech_prob": 0.5840025544166565,
      "words": [
        {
          "text": "Est-ce",
          "start": 35.02,
          "end": 35.12
        },
        {
          "text": "que",
          "start": 35.12,
          "end": 35.22
        },
        {
          "text": "vous",
          "start": 35.22,
          "end": 35.32
        },
        {
          "text": "allez",
          "start": 35.32,
          "end": 35.42
        },
        {
          "text": "bien?",
          "start": 35.42,
          "end": 35.58
        }
      ]
    }
  ],
  "language": "fr"
}